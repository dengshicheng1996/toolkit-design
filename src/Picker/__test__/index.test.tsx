import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import {
  unstable_Picker as Picker,
  unstable_List as List,
} from '@ant-design/mobile'

const seasons = [
  [
    {
      label: '2013',
      value: '2013',
    },
    {
      label: '2014',
      value: '2014',
    },
  ],
  [
    {
      label: '春',
      value: '春',
    },
    {
      label: '夏',
      value: '夏',
    },
  ],
]

describe('Picker', () => {
  it('snapshot', () => {
    const component = shallow(
      <div>
        <List>
          <Picker title="选择季节" data={seasons} cascade={false}>
            {/* <List.Item arrow="horizontal"> */}
            <h3 key="123">Multiple</h3>
            {/* </List.Item> */}
          </Picker>
        </List>
      </div>,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
