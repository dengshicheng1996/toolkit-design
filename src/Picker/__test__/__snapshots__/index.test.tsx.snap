// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picker snapshot 1`] = `
<div>
  <List>
    <Picker
      cascade={false}
      data={
        Array [
          Array [
            Object {
              "label": "2013",
              "value": "2013",
            },
            Object {
              "label": "2014",
              "value": "2014",
            },
          ],
          Array [
            Object {
              "label": "春",
              "value": "春",
            },
            Object {
              "label": "夏",
              "value": "夏",
            },
          ],
        ]
      }
      title="选择季节"
    >
      <h3
        key="123"
      >
        Multiple
      </h3>
    </Picker>
  </List>
</div>
`;
